lighthouse-cli/test/cli/performance-experiment/database-test.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-env mocha */
+const assert = require('assert');
+
+const PerfXDatabase = require('../../../performance-experiment/experiment-database/database');
+const sampleResults = require('../../../../lighthouse-core/test/results/sample');
+
+describe('Perf-X Database', function() {
+  it('can store and output experiments data', () => {
+    const perfXDatabase = new PerfXDatabase();
+    process.on('exit', () => perfXDatabase.clear());
```

Comment:
> i'd put this in an afterEach block (some other tests have them)

I didn't know that! Thx for your advice

> and do you want to use a fresh database for each of these test cases?

Are you referring to `testCases` or 'it'? I want to test the whether database is capable of storing multiple data sets here. I will rename `testCases` to `dataSets` to make it clear.